### ISSUE TITLE
v3.0.0 is not on PyPI

### ISSUE THREAD

**Issue by rtdev-com** [jazzband/django-payments#390]
`v3.0.0` does not exist via `pip install`.

```
ERROR: Could not find a version that satisfies the requirement django-payments[stripe]>=3.0.0 (from versions: 0.2, 0.2.1, 0.3, 0.3.1, 0.3.2, 0.3.3, 0.3.4, 0.3.4.1, 0.3.4.2, 0.3.4.3, 0.3.4.4, 0.4, 0.4.1, 0.4.2, 0.4.2.1, 0.4.2.2, 0.4.3, 0.5.0, 0.5.1, 0.5.2, 0.5.3, 0.5.3.1, 0.5.4, 0.6.0, 0.6.1, 0.6.2, 0.6.3, 0.6.4, 0.6.5, 0.6.6, 0.7.0, 0.7.1, 0.7.2, 0.7.3, 0.8.0, 0.8.1, 0.8.2, 0.8.3, 0.8.4, 0.9.0, 0.9.1, 0.9.2, 0.9.3, 0.9.4, 0.9.5, 0.9.6, 0.10.0, 0.11.0, 0.11.0.1, 0.11.0.2, 0.11.0.3, 0.12.0, 0.12.1, 0.12.2, 0.12.3, 0.13.0, 0.14.0, 0.15.0, 0.15.1, 1.0.0, 2.0.0)
ERROR: No matching distribution found for django-payments[stripe]>=3.0.0
```

https://pypi.org/project/django-payments/#history

![image](https://github.com/jazzband/django-payments/assets/81729714/c4b534b2-5ffb-404f-891c-974b6ea2bd00)

**Comment by WhyNotHugo** [comment]
The latest version is 2.0.0.

**Comment by rtdev-com** [comment]
Is there a doc that describes your release strategy? I am confused on when and which versions are released. 🤔

**Comment by hypy13** [comment]
I install from git using:
pip install git+https://github.com/jazzband/django-payments.git

**Comment by rtdev-com** [comment]
Nice, thank you @hypy13 and @WhyNotHugo.

**Comment by ahn-nath** [comment]
@rtdev-com 
I respectfully disagree with closing this as 'completed.' The new version should be made available through PyPI, or at the very least, the documentation should clearly specify version compatibility and provide consistent instructions. The current state is causing confusion.

I encountered this issue while following the Stripe integration documentation: The instructions reference a StripeProviderV3 class, but this class is not present in the latest version available through pip. This inconsistency led to the error "module 'payments.stripe' has no attribute 'StripeProviderV3'."

**I request the following:**

- Make the new version compatible with PyPI for easy installation.
- Update the documentation to accurately reflect available classes and versions.
- Provide clear guidance on version selection and installation methods.
- Resolving these inconsistencies will significantly improve the integration experience for developers.

**Comment by WhyNotHugo** [comment]
The documentation website should redirect to the last stable release by default. It currently redirects to the docs for latest main.

I’ll reopen this since that ought to be addressed.

**Comment by rtdev-com** [comment]
Reopening this issue

> The new version should be made available through PyPI, or at the very least, the documentation should clearly specify version compatibility and provide consistent instructions. The current state is causing confusion.

> The documentation website should redirect to the last stable release by default. It currently redirects to the docs for latest main.

**Comment by PaulleDemon** [comment]
Hi, any updates on releasing the latest to PYPI? There are some critical bugs in V2 preventing the payment flow,  seems to have been corrected in V3

**Comment by TakumiHendricksDev** [comment]
Is the PyPI version going to be updated? Its been almost 2 months since this has been addressed...

**Comment by WhyNotHugo** [comment]
I've tagged a new release, but GitHub isn't running the CI workflow t publish it.

`.github/workflows/release.yml` looks correct to me.

**Comment by WhyNotHugo** [comment]
Published
